x/auth/client/cli/tx_multisign.go

```diff
@@ -196,10 +189,208 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 	}
 }
 
+func GetMultiSignBatchCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "multisign-batch [file] [name] [[signature-file]...]",
+		Short: "Assemble multisig transactions in batch from batch signatures",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`SAssemble a batch of multisig transactions generated by batch sign command.
+
+Read signature(s) from [signature] file(s), generates multisig signatures compliant to the
+multisig key [name], and attach it to the transactions read from [file].
+
+Example:
+$ %s tx multisign-batch transactions.json k1k2k3 k1sigs.json k2sigs.json k3sig.json
```

Comment:
```suggestion
$ %s tx multisign-batch transactions.json multisigk1k2k3 k1sigs.json k2sigs.json k3sig.json
```

Reply:
it's a symbolic notation of multisig created from k1,k2,k3

Reply:
I know, I was proposing a more explicit name.

Reply:
yes, I updated it as it was more explicit